app.go

```diff
@@ -602,26 +602,16 @@ func (app *app) runShell(s string, args []string, prefix string) {
 
 }
 
-func (app *app) runPagerOnText(text io.Reader) {
+func (app *app) runPager(stdin io.Reader) {
```

Comment:
I'd call this `runPagerOn`, but I'll leave it up to you.

Reply:
So regarding the name changes, I felt it was wrong to use the word `text` since the source could theoretically be anything that produces bytes to be read. But I guess I don't have much opinion between `runPager` or `runPagerOn` so I'll just change it to your suggestion.